cscs-checks/prgenv/mpi.py

```diff
@@ -0,0 +1,46 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.required_version('>=2.14')
+@rfm.parameterized_test(['single'], ['funneled'], ['serialized'], ['multiple'])
+class Mpi_InitTest(rfm.RegressionTest):
+    def __init__(self, required_thread):
+        super().__init__()
+        self.descr = 'MPI_Init_thread ' + required_thread
```

Comment:
Delete the `self.descr` the default one generated by the parameterized test is ok.

Reply:
3122756caa398ff5c756822db5e54276c0e8bae1